homeassistant/helpers/script.py

```diff
@@ -719,7 +725,11 @@ async def _async_step_choose(self) -> None:
                                 await self._async_run_script(script)
                                 return
                     except exceptions.ConditionError as ex:
-                        _LOGGER.warning("Error in 'choose' evaluation:\n%s", ex)
```

Comment:
Similar to other conditions I suspect this line isn't reachable either, but I don't know with certainty if async_run_script could throw a ConditionError